flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/functions/utils/UserDefinedFunctionUtils.scala

```diff
@@ -162,24 +191,107 @@ object UserDefinedFunctionUtils {
   }
 
   /**
+    * Internal method of [[ScalarFunction#getResultType()]] that does some pre-checking and uses
```

Comment:
This method is never used. I carelessly introduced this method before. So I will remove this.